src/Effect.ss

```diff
@@ -26,13 +26,15 @@
   (define untilE
     (lambda (f)
       (lambda ()
-        (error #f "Effect:untilE not implemented"))))
+        (do () ((f))
+          '()))))
```

Comment:
The value of `unit` is the symbol [`'unit`](https://github.com/purescm/purescript-prelude/blob/v6.0.1-scm/src/Data/Unit.ss#L7)